prody/proteins/header.py

```diff
@@ -1111,10 +1111,10 @@ def buildBiomolecules(header, atoms, biomol=None):
             translation[2] = line2[3]
             t = Transformation(rotation, translation)
 
-            newag = atoms.select('chain ' + ' or chain '.join(mt[times*4+0])).copy()
+            newag = atoms.select('chain ' + ' or chain '.join(mt[times*4+0]))
             if newag is None:
                 continue
-            newag.all.setSegnames(decToHybrid36(times+1,resnum=True))
```

Comment:
Okay, I see now this is where you fix the selection problem.  I assume you know better than I do if it is okay to not call setSegnames.  However, there still will be problems with the selection "chain bb" when building a biomolecule.  See 7M57

Reply:
Can you explain why please?

Reply:
I was thinking that probably I do still need to set segnames but I need to check